skimage/morphology/convex_hull.py

```diff
@@ -155,6 +155,7 @@ def convex_hull_object(image, *, connectivity=2):
     ----------
     image : (M, N) ndarray
         Binary input image.
+
```

Comment:
I think this extra space is unexpected?

Reply:
It does not cause any problem in the rendered doc: https://1635-2014929-gh.circle-artifacts.com/0/doc/build/html/api/skimage.morphology.html#skimage.morphology.convex_hull_object

Reply:
I interpreted it as a convention to split between arguments before * and after. Just a guess that I found elegant.

Reply:
I don't believe there are written rules about this; wether to put blank lines between items varies across projects; currently my auto formatter has some heuristics, in particular if one items description have multiple paragraph (which is the case here for connectivity) it will add one line between all items.

It is quite hard to no modify whitespace as is does Docstring -> AST-Like repr -> back to text, tso initial whitespace information is lost.